ci-operator/config/openshift/bushslicer/openshift-bushslicer-master.yaml

```diff
@@ -0,0 +1,51 @@
+base_images:
+  base:
+    cluster: https://api.ci.openshift.org
+    name: plugin-base
+    namespace: bushslicer
+    tag: latest
+binary_build_commands: make all
```

Comment:
We do not use make. I believe nothing needs to be built in our image. It serves as a base image for QE Jenkins slaves. At the end it is stripped out of the bushslicer repo. If tests are to be run off this, then we need a new dockerfile where BushSlicer repo will be present.

Reply:
I'll remove the make commands.